client/src/components/Cell/CellMain.tsx

```diff
@@ -36,12 +40,32 @@ const CellMain = ({
   const Confirm = useConfirm();
   const Alert = useAlert();
   const { brute, owner } = useBrute();
-
+  const { user } = useAuth();
+  const navigate = useNavigate();
   const xpNeededForNextLevel = useMemo(
     () => (brute ? getXPNeeded(brute.level + 1) : 0),
     [brute],
   );
-
+  const arrowProps = {
+    maxHeight: '70px', maxWidth: '70px', color: '#7d4736', cursor: 'pointer', fontSize: '150px', margin: 'auto'
```

Comment:
As this variable never changes, you need to get it out of the component declaration, so that it doesn't retrigger on every render. (place it above instead).
No need for maxWidth/maxHeight, you can adjust the size of icons using the size attribute on the Fab component.
You should never use specific colors in this app, use theme variable instead.